docs/macros_unit_tests.md

```diff
@@ -0,0 +1,42 @@
+{% docs unit_test_description %}
+    describes the atomic unit test that need to be run:
```

Comment:
This macro describes an atomic unit test that is executed when running 'dbt test'

Reply:
done